packages/suite/src/support/suite/styles/GlobalStyleProvider.tsx

```diff
@@ -55,10 +52,4 @@ const GlobalStyle = createGlobalStyle<{ theme: SuiteThemeColors; themeVariant: S
     ${notifications}
 `;
 
-const GlobalStyleProvider = () => {
-    const { theme, themeVariant } = useTheme();
-
-    return <GlobalStyle theme={theme} themeVariant={themeVariant} />;
-};
-
-export default GlobalStyleProvider;
+export default GlobalStyle;
```

Comment:
maybe rename the file because it's no longer the provider

Reply:
Technically it provides exactly the same thing as before, but now you have to tell it from outside for which theme. Nevertheless, agreed and fixed in https://github.com/trezor/trezor-suite/pull/4314/commits/288b79bb7e9fd4bcaa1688da118ceb8ab19277b8.